Automattic/wp-calypso
# Store: Stripe Connect: Add notification of account connection completion

Fixes #19297 

To test:
* First, to test this on your local machine, you **must** set your site to use the staging woocommerce services server. To do so, disconnect any stripe account you have connected to your site and then install and activate the "woocommerce services use staging" plugin from https://github.com/automattic/
* Next, install this branch and make sure `npm run test-client /client/extensions/woocommerce/` passes (new tests were added for a new flag in redux)
* Next, navigate to http://calypso.localhost:3000/store/{DOMAIN}
* Click on Manage for Stripe
* If you get the dialog that shows keys, click on "I want to use Stripe Connect instead" link at the bottom of the dialog
* Select "Create new" or "I have an existing" as appropriate for you
* If you selected "I have an existing", authorize the connection at Stripe
* Wait while the connection is being completed
* Bask in the glory of the new notice
* Try the dismiss affordance
* Get rid of the connection and the staging plugin
* Profit!

FWIW, it looks like this after the user has completed creating/connecting their account -- if their Stripe account still needs to be activated:

<img width="740" alt="connect-success-needs-activation" src="https://user-images.githubusercontent.com/1595739/35593016-98bf147c-05c3-11e8-83ab-cbaa82586bad.png">

Or... if their Stripe account is already activated:

<img width="881" alt="connect-success-and-already-activated" src="https://user-images.githubusercontent.com/1595739/35593014-98a41582-05c3-11e8-977d-e464b764ff66.png">


Design notes: Grammar was tweaked a bit. Standard Notice styling (instead of full bleed in original mocks) ok'd by @kellychoffman 

cc @kellychoffman @jameskoster
